src/components/Footer/Footer.js

```diff
@@ -8,7 +8,17 @@ export const StyledFooter = styled.div`
   height: ${(props) => props.height || '15%'};
   justify-content: space-between;
   margin: ${(props) => props.margin || '0px'};
-  background-color: blue;
+  /* background-color: red; */
+  padding: ${(props) => props.padding || '0px'};
+  margin-top: auto;
+
+  @media (max-width: 600px) {
```

Comment:
Do you have s list of different widths for responsive?
Maybe you can make 600px as a constant

Reply:
The max-width here should have been 556px

but  I do have a constant of 556px in the Navbar.style.js

I will export it to make it accessible to this file